README.md

```diff
@@ -1,3 +1,11 @@
+<!--
+SPDX-FileCopyrightText: 2021 Frieder Schrempf
+SPDX-FileCopyrightText: 2021 Carl Schwan <carl@carlschwan.eu>
+SPDX-FileCopyrightText: 2021 Cyrille Bollu
+SPDX-FileCopyrightText: 2021 Pierre Ozoux <pierre@ozoux.net>
+SPDX-FileCopyrightText: 2021 Julius Härtl <jus@bitgrid.net>
+SPDX-License-Identifier: AGPL-3.0-or-later
+-->
```

Comment:
Are these tags auto-generated from the git history?
Why are some e-mail addresses missing?
Do we really need copyright and license information in documentation files such as `README.md`? If yes is AGPL the correct license for documentation files or should they be CC0 or something else?

Reply:
Would also be curious what the way would be to update this automatically from the git history. :)